hw/dv/sv/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: "SV"
+title: "SystemVerilog"
```

Comment:
I would call this `Common SystemVerilog / UVM components`.

Reply:
Thanks, I've changed this.